pkg/runner/local_docker.go

```diff
@@ -70,6 +70,12 @@ type LocalDockerRunnerConfig struct {
 	// (default: ["nofile=1048576:1048576"]).
 	Ulimits []string `toml:"ulimits"`
 
+	// EnvironmentVariables can be used to set additional environment variables.
+	// Note to use unique environment variables not used by Testground,
+	// as your custom environment variables will get overwritten by the ones
+	// also set by Testground.
+	EnvironmentVariables map[string]string `toml:"environment_variables"`
```

Comment:
Thanks for the clear documentation,

Could you rename it to `AdditionalEnvs`? It'll be explicit that these are "bonus" env variables, which don't overwrite Testground's and will be consistent with `AdditionalHosts`

Reply:
Alright. Done :)